src/libConsensus/ConsensusLeader.cpp

```diff
@@ -177,20 +220,29 @@ void ConsensusLeader::StartConsensusSubsets() {
   }
 
   m_numSubsetsRunning = m_consensusSubsets.size();
-  for (unsigned int index = 0; index < m_consensusSubsets.size(); index++) {
+  // subset 0 last to be started. giving community nodes the advantage
+  for (unsigned index = m_consensusSubsets.size(); index > 0; index--) {
```

Comment:
Change `unsigned` to `unsigned int` to be more explicit

Reply:
ok.